db/migrate/20160414035822_create_identities.rb

```diff
@@ -2,6 +2,7 @@ class CreateIdentities < ActiveRecord::Migration
   def change
     create_table :identities do |t|
       t.references :user, index: true, foreign_key: true
+      t.references :brand, index: true, foreign_key: true
```

Comment:
You shouldn't alter an existing migration especially if that migration has already been run in prod. Let's change this to be an additive migration.